code/modules/research/designs/medical_designs.dm

```diff
@@ -132,10 +132,10 @@
 	category = list("Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_ALL
 
-/datum/design/crewpinpointer
+/datum/design/pinpointer_crew
```

Comment:
is this intentional

Reply:
mostly to bring it in line with pinpointer_tracker because trackerpinpointer or trackingpinpointer is just a mouthful of words, it's not necessary outside of looking better in my opinion as well as having consistency